acinclude.m4

```diff
@@ -1,10 +1,10 @@
 
-# QL_CHECK_CPP11
+# QL_CHECK_CPP14
 # --------------------
-# Check whether C++11 features are supported by default.
-# If not (e.g., with Clang on Mac OS) add -std=c++11
-AC_DEFUN([QL_CHECK_CPP11],
-[AC_MSG_CHECKING([for C++11 support])
+# Check whether C++14 features are supported by default.
+# If not (e.g., with Clang on Mac OS) add -std=c++14
+AC_DEFUN([QL_CHECK_CPP14],
+[AC_MSG_CHECKING([for C++14 support])
  AC_COMPILE_IFELSE(
     [AC_LANG_PROGRAM(
         [[@%:@include <initializer_list>
```

Comment:
We need to add some C++14 features in here, otherwise a compiler that only supports C++11 will pass the check.

Reply:
Done